aioshelly/block_device.py

```diff
@@ -268,6 +268,17 @@ async def trigger_shelly_gas_unmute(self) -> None:
         """Trigger a Shelly Gas unmute action."""
         await self.http_request("get", "unmute")
 
+    async def enable_shelly_motion_detection(self, enable: bool) -> None:
```

Comment:
```suggestion
    async def set_shelly_motion_detection(self, enable: bool) -> None:
```
To be consistent with other set commands we have

Reply:
OK, changed, thank you.